pkg/beholder/config.go

```diff
@@ -0,0 +1,52 @@
+package beholder
+
+import (
+	"time"
+
+	otelattr "go.opentelemetry.io/otel/attribute"
+)
+
+type Config struct {
+	InsecureConnection       bool
+	CACertFile               string
+	OtelExporterGRPCEndpoint string
+
+	// OTel Resource
+	ResourceAttributes []otelattr.KeyValue
+	// Message Emitter
+	EmitterExportTimeout time.Duration
+	// OTel Trace
+	TraceSampleRate   float64
```

Comment:
This is called `Ratio` in the otel api and our other existing options:
```suggestion
	TraceSampleRatio   float64
```